content/spaces/setup.textile

```diff
@@ -0,0 +1,79 @@
+
+Both SDKs are available as "NPM modules":#npm and via "CDN.":#cdn
+
+<aside data-type='important'>
+<p>The examples use "basic authentication":/auth/basic to demonstrate features for convenience. In your own applications, basic authentication should never be used on the client-side as it exposes your Ably API key. Instead use "token authentication.":/auth/token</p>
+</aside>
+
+h3(#npm). Using NPM
+
+Install the Ably JavaScript SDK and the Spaces SDK:
+
+```[sh]
+npm install ably @ably/spaces
+```
+
+Import the SDKs into your file:
+
+```[javascript]
+import Spaces from '@ably/spaces';
+import { Realtime } from 'ably';
+```
+
+Instantiate a realtime client using the Ably JavaScript SDK. Pass the generated client into the Spaces constructor:
+
+```[javascript]
+const client = new Realtime.Promise({key: "<API-key>", clientId: "<client-ID>"});
+const spaces = new Spaces(client);
+```
+
+A "@ClientOptions@":/api/realtime-sdk#client-options object may be passed to the Ably JavaScript SDK to further customize the connection, however at a minimum you must set your API key and provide a @ClientId@ so that the client is "identified.":/auth/identified-clients
+
+h3(#cdn). Using a CDN
+
+Reference the Ably JavaScript SDK and the Spaces SDK within the @<head>@ of your HTML file:
+
+```[html]
+<script src="https://cdn.ably.com/lib/ably.min-1.js"></script>
+<script src="https://cdn.ably.com/spaces/0.1.0/iife/index.bundle.js"></script>
+```
+
+Instantiate a realtime client using the Ably JavaScript SDK. Pass the generated client into the Spaces constructor:
+
+```[javascript]
+const client = new Realtime.Promise({key: "<API-key>", clientId: "<client-ID>"});
+const spaces = new Spaces(client);
+```
+
+A "@ClientOptions@":/api/realtime-sdk#client-options object may be passed to the Ably JavaScript SDK to further customize the connection, however at a minimum you must set your API key and provide a @ClientId@ so that the client is "identified.":/auth/identified-clients
+
+h2(#connections). Client connections
+
+A Spaces client exposes the underlying "connection":/connect to Ably that is established via the Ably JavaScript SDK. This means that Spaces clients benefit from the same functionality available in the Ably JavaScript SDK, such as automatic transport selection and "connection state recovery":/connect/states in the event of brief disconnections.
+
+Connections transition through multiple states throughout their lifecycle. Whilst these transitions are handled by the Ably SDK, there are certain cases where you may need to observe and handle them within your application. Ably SDKs enable these transitions to be observed and triggered using methods available on the @connection@ object. The Spaces SDK exposes the underlying connection with @spaces.connection@, which is a reference to "@client.connection@":/api/realtime-sdk/connection in the Ably JavaScript SDK.
```

Comment:
In Dom's docs PR for simple connection management, there's also info about accessing the ably client via spaces too. I think we should mention it on this page.